init-openstack.html.md.erb

```diff
@@ -151,28 +151,36 @@ You must create and configure two Security Groups to restrict incoming network t
     $ git clone https://github.com/cloudfoundry/bosh-deployment
 
     # Fill below variables (replace example values) and deploy the Director
+    $ DIRECTOR_NAME='bosh-1'     # arbitrary name for the director
```

Comment:
One thing I could do (which I personally do not like that much):
```
command \
    `# --first is very important` \
    --first arg \
    `# --second is super important too` \
    --second arg
```
Any better way to do inline comments properly?